### PR TITLE
Fixed merge conflicts and restore Google user sign in

### PR DESCRIPTION
**Pull request by rs929**
## Overview

- Fixed merge conflicts with PRs
- Implemented restore Google user sign in

## Changes Made

- Addressed compile errors after merging
- Fixed restore Google Sign in to pass access token

## Next Steps (delete if not applicable)

- Implement loading screen for opening app

### CODE REVIEW

**Review by tiffany-pann — APPROVED** [review]
LGTM 🚀

File: Scoop/SceneDelegate.swift [hunk 1]
```diff
@@ -88,4 +109,15 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         UINavigationBar.appearance().backIndicatorTransitionMaskImage = backArrowImage
         UIBarButtonItem.appearance().setBackButtonBackgroundImage(transparentImage, for: .normal, barMetrics: .default)
     }
+    
+    private func setNetID(email: String) {
+        guard let strIndex = email.firstIndex(of: "@") else{
```
Comment (ReadeP) [inline Scoop/SceneDelegate.swift]: spacing `else {`

File: Scoop/SceneDelegate.swift [hunk 2]
```diff
@@ -88,4 +109,15 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
...
+            print("Not a valid Cornell email")
+            return
+        }
+        let index = email.index(strIndex, offsetBy: -1)
+        let netID = String(email[...index])
+        NetworkManager.shared.currentUser.netid = netID
```
Comment (ReadeP) [inline Scoop/SceneDelegate.swift:120]: Indenting looks off here

File: Scoop/SceneDelegate.swift [hunk 3]
```diff
@@ -43,7 +43,28 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
...
+            NetworkManager.shared.authenticateUser(googleID: userID, email: email, firstName: firstName, lastName: familyName, id_token: idtoken) { result in
+                switch result {
+                case .success(let user):
+                    self.setNetID(email: email)
+                    NetworkManager.shared.currentUser.firstName = firstName
+                    NetworkManager.shared.currentUser.lastName = familyName
+                    NetworkManager.userToken = user.accessToken
+                    self.didCompleteLogin()
+                case .failure:
+                    print("Unable to Authorize")
```
Comment (ReadeP) [inline Scoop/SceneDelegate.swift]: Can we print the error message as well for debugging sake?

**Review by ReadeP — APPROVED** [review]
Looks good!

**Pull request merged**